### PR TITLE
Text Stroke

### LINKED ISSUE #41: Too little vertical padding on Table Cell
**Describe the bug**
There is too little padding on Table Cells which makes Table harder to read.

**Expected behavior**
There should be a bit more vertical padding.

**Screenshots**
image

**Browser**
Edge 99.0.1150.46

### LINKED ISSUE #42: Layout Grid Cols has weird behavior
**Describe the bug**
There is too much row gap which makes the items look disconnected.
On mobile, items line up horizontally and do not even wrap, which makes accessing content that should have been immediately visible confusing.

**Expected behavior**
Row gap should be less than column gap, just like in Dialog Section.
Items should stack vertically on mobile, not line up horizontally.

**Screenshots**
image

**Browser**
Edge 99.0.1150.46

**Additional context**
There is also only one util modifier for this. Maybe look into that.

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
**Features**
- Text Stroke utils (adds outline to text)

**Fixes**
- https://github.com/suankularb-wittayalai-school/sk-components/issues/41
- https://github.com/suankularb-wittayalai-school/sk-components/issues/42

**Refs**
Fixes #41, fixes #42

### CODE REVIEW

File: src/scss/utils/text-stroke.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// Mixin for internal use
+// prettier-ignore
+@mixin text-stroke($color, $width: 2px) {
+  overflow: initial;
+  text-shadow:
+    calc(-1 * $width) calc(-1 * $width) 0 $color,
```
Comment (Jimmy-Tempest) [inline src/scss/utils/text-stroke.scss:6]: I have no idea what this does

**Review by Jimmy-Tempest — APPROVED** [review]
Look mostly understandable to me!

File: src/scss/utils/text-stroke.scss (hunk 1 above)
Comment (SiravitPhokeed) [reply src/scss/utils/text-stroke.scss:6]: This is a mixin adaptation of [this answer from StackOverflow](https://stackoverflow.com/questions/4919076/outline-effect-to-text#4919231), which is basically a workaround of the experimental webkit property called [text-stroke](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke).

**Review by IHasDiabetes — APPROVED** [review]
LGTM

**Pull request merged**